### ISSUE TITLE
Extend Documentation

### ISSUE THREAD

**Issue by pwfraley** [CodeDredd/pinia-orm#1753]
### Describe the feature

It would be nice if the documentation (api) would also describe why a function exists, which itch it scratches and when to use it.

### typescript

it would be nicer if:

## load()
This function is for loading data, it is used, when fetching via api is not possible.

### Additional information

- [ ] Would you be willing to help implement this feature?

### Final checks

- [X] Check existing [discussions](https://github.com/CodeDredd/pinia-orm/discussions) and [issues](https://github.com/CodeDredd/pinia-orm/issues).

**Comment by CodeDredd** [comment]
Fair point — the API pages mostly show *how* and skip the *why/when*. I'm extending the docs alongside the 2.0.0 release (several pages already got usage context recently). If you have concrete functions besides `load()` where the purpose is unclear, list them here — that helps prioritizing which pages get the treatment first.
